pyproject.toml

```diff
@@ -7,3 +7,51 @@ disable_error_code = ["union-attr", "annotation-unchecked", "import-untyped"]
 explicit_package_bases = true
 warn_unused_ignores = true
 exclude = ['tests']
```

Comment:
why excluding tests from the PyPI tarball?

Reply:
I thought we would only need the tests when developing.